p2p/ipld/plugin/go.mod

```diff
@@ -0,0 +1,13 @@
+module github.com/lazyledger-core/p2p/ipld/plugin
+
+go 1.15
+
+require (
+	github.com/ipfs/go-block-format v0.0.2
+	github.com/ipfs/go-cid v0.0.7
+	github.com/ipfs/go-ipfs v0.7.0
+	github.com/ipfs/go-ipfs-api v0.2.0
+	github.com/ipfs/go-ipld-format v0.2.0
+	github.com/lazyledger/nmt v0.0.0-20210113105746-8340e76c6def
```

Comment:
after https://github.com/lazyledger/nmt/pull/16 gets merged, this needs to be updated (currently, we use this branch)